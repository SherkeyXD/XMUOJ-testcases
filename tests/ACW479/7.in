11
17 20 58 44 53 40 79 8 57 50 37
