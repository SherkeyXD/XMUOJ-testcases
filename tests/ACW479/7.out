712690243
1 8 6 4 2 3 5 7 10 9 11 
